Library/Formula/ffe.rb

```diff
@@ -0,0 +1,17 @@
+class Ffe < Formula
+  homepage "http://ff-extractor.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/ff-extractor/ff-extractor/0.3.4/ffe-0.3.4.tar.gz"
+  sha1 "9e0f435568a65ef552d71e1a10a12f0627e1ccf9"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/ffe", "-V"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!